### ISSUE TITLE
OpenColorIO Project Review- 10/18

### ISSUE THREAD

**Issue by yarille** [AcademySoftwareFoundation/tac#474]

**Comment by yarille** [comment]
Pending email vote for renewal due to lack of quorum. @carolalynn22 to send information for email.

**Comment by jmertic** [comment]
@yarille to send out vote for renewal; no need for TAC to approve WG within OpenColorIO

**Comment by yarille** [comment]
Renewal approved via email TAC vote (1/16/2024)

**Comment by carolalynn** [comment]
Hi getting ahead of this - will this fall on Nov 13? Just trying to plan around travel etc... :)

**Comment by jmertic** [comment]
Yes - I've corrected the date now. Let @yarille  know if that is a good date for you.

**Comment by carolalynn** [comment]
We need to reschedule this, sorry. I need to figure out a better way to make sure I get notifications about these issues...

**Comment by jmertic** [comment]
Presentation on 2025-03-19 - voting vie LFX Voting

**Comment by michellearoth** [comment]
TAC voted to approve renewal of OpenColorIO at adopter stage.
